C/sort/counting sort.c

```diff
@@ -0,0 +1,61 @@
+/* Counting sort is a sorting algorithm that sorts the elements of an array by 
+counting the number of occurrences of each unique element in the array. 
+The count is stored in an auxiliary array and the sorting is done by 
+mapping the count as an index of the 
+auxiliary array.
+# The average time complexity for Counting Sort is O(n + k). 
+  The worst time complexity is O(n + k).
+# The best complexity for Counting Sort is O(n + k). */
+
+/* C Program to sort Elements Using Counting Sort */
+#include <stdio.h>  
+ 
+/*  Counting sort function  */
+void counting_sort(int A[], int k, int n)
+{
+    int i, j;
+    int B[15], C[100];
```

Comment:
Do not fix the size of the arrays, calculate the max size of the count array using some algorithm.